data_diff/diff_tables.py

```diff
@@ -142,7 +159,7 @@ def count_and_checksum(self) -> Tuple[int, int]:
         """Count and checksum the rows in the segment, in one pass."""
         start = time.time()
         count, checksum = self.database.query(
-            self._make_select(columns=[Count(), Checksum(self._relevant_columns)]), tuple
+            self._make_select(columns=[Count(), Checksum(self._relevant_columns_repr)]), tuple
```

Comment:
What do you think about in `debug` mode, we output `concat` value generated by the database? Seems really useful to debug the types of issues you've run into. So you see something like for `id=2`, `time=2020-01-01 00:00:00.000000`:

```
CHECKSUMMED VALUE: 22020-01-01 00:00:00.000000
```

Reply:
Maybe, yeah.